.php-cs-fixer.dist.php

```diff
@@ -4,6 +4,7 @@
 
 $finder = (new PhpCsFixer\Finder())
     ->in(__DIR__)
+    ->ignoreDotFiles(false)
```

Comment:
the `.twig-cs-fixer.php` and `.php-cs-fixer.dist.php` where currently ignored but should be included. The VCS files `.git` is still ignored via ignoreVCS.